resource-managers/mesos/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosCoarseGrainedSchedulerBackend.scala

```diff
@@ -637,7 +637,7 @@ private[spark] class MesosCoarseGrainedSchedulerBackend(
       if (state.equals(TaskState.RUNNING) &&
           shuffleServiceEnabled &&
           !slave.shuffleRegistered) {
-        assume(mesosExternalShuffleClient.isDefined,
```

Comment:
@JoshRosen . Since this is not a test case, could you remove `[TESTS]` in the PR title?

Reply:
I've rolled back this non-test change, so PR is now test-only.